core/winnt/src/TWinNTSystem.cxx

```diff
@@ -4319,10 +4319,12 @@ const char *TWinNTSystem::GetLibraries(const char *regexp, const char *options,
          // full path not found, so split it to extract the library name
          // only, set its extension to '.lib' and add it to the list of
          // libraries to search
```

Comment:
Can you update the comment to reflect the "path must not contain `C:\\WINDOWS`?

Reply:
Will do